packages/components/src/components/InternalHeader/InternalHeader.types.tsx

```diff
@@ -6,11 +6,11 @@ import { type SvgIcon } from '../Icon/utils';
 
 type NavigationLinkProps = {
   href: string;
-  title: string;
+  children: string;
```

Comment:
Nå som vi går mer mot en generell `children`, burde vi også gjøre om typen til `ReactNode`? Kanskje mer forventet adferd.

```suggestion
  children: ReactNode;
```

Reply:
Jeg tenkte på det, men vi har også `icon` prop, så konsumenten setter ikke alle barna via `children` prop. Men du har nok rett, det bør være mulig å ha mer enn tekst, hvis man f.eks. trenger usynlig tekst osv. Kan bytte til `ReactNode` og skrive i prop-beskrivelsen at ikon settes med egen.